athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/glue/DefaultGlueType.java

```diff
@@ -71,10 +74,40 @@
         this.arrowType = arrowType;
     }
 
-    public static DefaultGlueType fromId(String id)
+    private static ArrowType getDecimalArrowType(String in)
     {
-        DefaultGlueType result = TYPE_MAP.get(id.toLowerCase());
+        Matcher decimalMatcher = decimalPattern.matcher(in);
+        if (!decimalMatcher.matches()) {
+            return null;
+        }
+        try {
+            int[] params = Arrays.stream(decimalMatcher.group(2).split(","))
```

Comment:
what does decimalMatcher.group(2) represent?

Reply:
It means the 2nd capture group (things surrounded by ()) in the regex.
See this: https://i.imgur.com/yLRFRAC.png